lib/deployinator/templates/log_table.mustache

```diff
@@ -3,7 +3,7 @@
         <tr>
             <th>
                 {{# prev_page }}
-                <form action="/log">
```

Comment:
otherwise, a page that uses that template such as `/some_stack/log`'s next / prev buttons direct to `/log`

whereas with this change, the next / prev buttons are directed to the same page but with different params